skycoin/skywire
# Mac installer

Did you run `make format && make check`? yes

For Skywire-Visor Release 0.5.0-rc*

 Changes:	
- Package skywire-visor as macos application

TODO:
- [x] Fix application not installed correctly 
- [x] Fix updating application
- [x] Fix application icon (not systray icon) not showing properly on `/Applications`
- [x] Also builds arm64 binaries

To Test:

- Run `make mac-installer` and run the installer via `open ./SkywireInstaller.pkg` or see https://github.com/skycoin/skywire/pull/889#issuecomment-925639466
